src/state/zustand/swapperStore/amountSelectors.ts

```diff
@@ -564,3 +583,13 @@ export const selectTradeAmountsByActionAndAmountFromQuote: Selector<
     }
   },
 )
+
+export const selectDonationAmountFiat = createSelector(
+  selectSellAmountFiat,
+  selectAffiliateBps,
+  (sellAmountFiat, affiliateBps): string | undefined => {
```

Comment:
this can strictly return a string too

Reply:
Addressed in https://github.com/shapeshift/web/pull/4353/commits/47b3c94b52c5a0bb478160ced5b91a2c9b764122.